### ISSUE TITLE
This project is dead?

### ISSUE THREAD

**Issue by jcbritobr** [rust-lang/rls-vscode#652]
I dont see any updates in this extension. Is this project already dead?

**Comment by Xanewok** [comment]
Nope! We're figuring out the technical side of things (marketplace publishing), otherwise it's alive and kicking!

**Comment by Extrawurst** [comment]
@Xanewok is that still true?
